DEVELOPER_GUIDE.md

```diff
@@ -80,6 +94,7 @@ Integration tests can be run on the following operating systems:
   * Amazon Linux 2
   * MacOS
 
+Invoke integration tests by running the following command within the root directory of the repository:
```

Comment:
Would be good to note the amount of time the user should expect to run the integ tests.